web/ui/dashboard/src/app/private/components/create-project-component/create-project-component.component.ts

```diff
@@ -272,14 +278,14 @@ export class CreateProjectComponent implements OnInit {
 	}
 
 	cancel() {
-		this.confirmModal = true;
+		this.confirmationDialog.nativeElement.showModal()
 		document.getElementById('projectForm')?.scroll({ top: 0, behavior: 'smooth' });
 	}
 
 	confirmToggleAction(event: any, actionType?: 'metaEvents' | 'endpoints') {
 		const disableValue = event.target.checked;
-		if (actionType !== 'metaEvents') disableValue ? this.updateProject() : (this.disableEndpointsModal = true);
-		else if (!disableValue && actionType === 'metaEvents') this.showMetaEventPrompt = true;
```

Comment:
Looks like we still have these modal display variables around: `disableEndpointsModal`, `showMetaEventPrompt`.